test/hotspot/jtreg/vmTestbase/nsk/jdi/Argument/value/value004.java

```diff
@@ -122,8 +122,7 @@ public static int run(String argv[],PrintStream out) {
                 }
                 flg = true;
                 ovl = argHandler.getLaunchExecPath() + " "
-                     + "-Xdebug -Xnoagent -Djava.compiler=NONE "
-                     + "-Xrunjdwp:transport=" + c.transport().name() + ",suspend=y,"
+                     +"-agentlib:jdwp=transport=" + c.transport().name() +",server=n,suspend=y,"
```

Comment:
Good. Full speed debugging came in JDK 1.4. In JDK 1.5, -agentlib and -agentpath were added a standard options for starting agents on the command line so -Xdebug because redundant. I think -Xnoagent dates from Classic VM.